# A command with a action to id the function and lists of PoseStamped and Topic arguments
int id
string action
geometry_msgs/PoseStamped[] ps_list
string[] topic_list
